Clinical-Genomics/scout
# Fix demo delivery report and delivery report orientation

Show a partial fix to #3108 

- Fixes missing delivery report for demo case in Scout
- Replaces delivery report file with a more recent file with additional cols and landscape iroentation 

<details>
<summary>Testing on cg-vm1 server (Clinical Genomics Stockholm)</summary>

**Prepare for testing**
1. Make sure the PR is pushed and available on [Docker Hub](https://hub.docker.com/repository/docker/clinicalgenomics/scout-server-stage)
1. `ssh your.email@cg-vm1.scilifelab.se`
1. `sudo -iu hiseq.clinical`
1. `ssh localhost`
1. (optional) Find out which scout branch is currently deployed on cg-vm1: `podman ps`
1. Stop the service with current deployed branch: `systemctl --user stop scout.target`
1. Start the scout service with the branch to test: `systemctl --user start scout@<this_branch>`
1. Make sure the branch is deployed: `systemctl --user status scout.target`
</details>

**How to test**:
1. Locally: on master branch, make sure the delivery report doesn't work.
2. Switch to this branch and make sure that delivery report works and it's exported in PDF format with all the text fitting the page (page doesn't have a logo)
3. Install on cg-vm1 and check that delivery reports are shown for existing cases (PDF report might not be fitting the page)

**Expected outcome**:
The functionality should be working
Take a screenshot and attach or copy/paste the output.

**Review:**
